### DISCUSSION TITLE
Get POST from Slack upon button press

### DISCUSSION THREAD

**Discussion by atauln** [slack-go/slack#1103] [Q&A]
Hi! I'm using the button example documentation to create a Slack message that has a button on it. While I got the button to work, how would I go about receiving the POST message from the server when it's pressed? The code 
`http.HandleFunc("/actions", buttonHandler)
http.ListenAndServe(":3000", nil)` 
never fires, even upon clicking the button within the message.

**Comment by kanata2** [comment]
Please give more details for your code, slack app configurations.
